clients/client-java/src/test/java/org/apache/gravitino/client/integration/test/authorization/AccessControlIT.java

```diff
@@ -184,6 +184,12 @@ void testManageRoles() {
     String[] roleNames = metalake.listRoleNames();
     Arrays.sort(roleNames);
 
+    Assertions.assertEquals(
+        Lists.newArrayList(anotherRoleName, roleName), Arrays.asList(roleNames));
+
+    // List roles by the object
+    roleNames = metalake.listBindingRoleNames();
+    Arrays.sort(roleNames);
```

Comment:
Can you test list roles for different objects other than metalek?

Reply:
Added the test cases of catalog, schema and fileset.